editor/block-switcher/style.scss

```diff
@@ -5,6 +5,7 @@
 	font-size: $default-font-size;
 	line-height: $default-line-height;
 	margin-right: -1px;
+	margin-bottom: -1px;
```

Comment:
Can you explain this change?

Reply:
This fixed a double border issue, but it looks like it's unnecessary after updating the remaining `ul` references.